<launch>
  <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="launch_prefix" default="" />

  <!-- configure camera input -->
  <arg name="camera_name" default="/usb_cam/" />
  <arg name="image_topic" default="image_raw" />
  <arg name="queue_size" default="1" />


  <node pkg="artag_nav_commander" type="livecam_nav_commander_node" name="livecam_nav_commander_node" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)" />
    <remap from="camera_info" to="$(arg camera_name)/camera_info" />

    <param name="publish_tag_detections_image" type="bool" value="true" /><!-- default: false -->
    <param name="queue_size" type="int" value="$(arg queue_size)" />

    <rosparam command="load" file="$(find artag_nav_commander)/config/settings.yaml"/>
    <rosparam command="load" file="$(find artag_nav_commander)/config/tags.yaml"/>
  </node>
</launch>
